Modules/TPC/include/TPC/Clusters.h

```diff
@@ -54,7 +54,8 @@ class Clusters /*final*/ : public TaskInterface // todo add back the "final" whe
   void reset() override;
 
  private:
-  o2::tpc::qc::Clusters mQCClusters{};                         ///< O2 Cluster task to perform actions on cluster objects
+  bool mIsMergeable = true;
+  ClustersData mQCClusters{ "N_Clusters" };                    ///< O2 Cluster task to perform actions on cluster objects
```

Comment:
```suggestion
  ClustersData mQCClusters{};                    ///< O2 Cluster task to perform actions on cluster objects
```
The default should be "N_Clusters" so this is not really needed.

Reply:
But it also doesn't hurt to leave it like this...

Reply:
done